splinterd/api/static/openapi.yaml

```diff
@@ -414,9 +414,12 @@ paths:
           content:
             application/json:
               schema:
-                type: array
-                items:
-                  type: '#/components/schemas/Permission'
+                type: object
```

Comment:
This change does not belong to this commit

Reply:
Fixed (moved to previous commit)